.github/workflows/wc_buid_binaries.yml

```diff
@@ -0,0 +1,54 @@
+name: build binaries
```

Comment:
shouldn't the file name should be `wc_build_binaries.yml`?

Reply:
Sorry, I don't follow since that's the file name. Or do you mean the `name` field as well? If that's so I'd rather go with a human reabable one.

Reply:
Oh! I see it now. Thank you, let me change that right away.

Reply:
Solved